.azure-pipelines/official-build.yml

```diff
@@ -15,8 +15,18 @@ trigger: none
 pr: none
 
 stages:
-- stage: Build
+- stage: Build_barefoot
```

Comment:
why are we having so many stages?

Reply:
Just for beter result showing. Here is an example: https://dev.azure.com/mssonic/build/_build?definitionId=72

Reply:
For [nephos](https://sonic-jenkins.westus2.cloudapp.azure.com/job/nephos/job/buildimage-nephos-all/), the build for master branch breaks for a month. If we keep them in one stage, the result is always fail.

Reply:
stage is designed to have dependency. if we want to make the build independent, then we can create multiple pipeline and each pipeline handle one platform.

Reply:
we can remove dependency by:   dependsOn: []